### ISSUE TITLE
Document what $$options.hunked is for

### ISSUE THREAD

**Issue by smoakey** [kamilkp/angular-vs-repeat#160]
While looking for a solution to "even/odd" styling referenced in this other issue https://github.com/kamilkp/angular-vs-repeat/issues/14 I ran across the param `$$options.hunked` in the code that seems to offer a fix to my problem.

The combination of `vs-excess="4"` and `vs-options="{hunked: true}"` added to my vs-repeat fixed my issue.

**Comment by tdecker91** [comment]
Yes some documentation would be nice. Thanks for posting about it here @smoakey this helped me a lot I was having the same exact issue
